src/modals/AddressSweepModal.tsx

```diff
@@ -139,10 +139,14 @@ const AddressSweepModal = ({ sweepAddress, onClose, onSuccessfulSweep }: Address
           id="to-address"
         />
         <InfoBox Icon={Info} contrast noBorders>
-          {t`This operation will sweep all funds from`}{' '}
-          <ColoredWord color={sweepAddresses.from.settings.color}>{sweepAddresses.from.getName()}</ColoredWord>{' '}
-          {t`and transfer them to`}{' '}
-          <ColoredWord color={sweepAddresses.to.settings.color}>{sweepAddresses.to.getName()}</ColoredWord>.
+          <Trans t={t} i18nKey="sweepOperationFromTo">
+            This operation will sweep all funds from
+            <ColoredWord color={sweepAddresses.from.settings.color}>
+              {{ from: sweepAddresses.from.getName() }}
+            </ColoredWord>
+            and transfer them to
+            <ColoredWord color={sweepAddresses.to.settings.color}>{{ to: sweepAddresses.to.getName() }}</ColoredWord>.
+          </Trans>
```

Comment:
I think this solution is more appropriate because there might be some languages where the sentences break at different points than in English, as the previous solution was implementing.

Reply:
Nice one.

Reply:
Very nice point :D